include/barcodes/pdf417.php

```diff
@@ -740,6 +740,8 @@ public function getBarcodeArray() {
 	 * @protected
 	 */
 	protected function getErrorCorrectionLevel($ecl, $numcw) {
+		$maxecl = 8; // starting error level
```

Comment:
Note this code here has several bugs and it is way behind:
https://github.com/tecnickcom/tc-lib-barcode
It would be better to switch to the new library.

Reply:
`$maxecl` must be declared before the `if` cond. of course ;-)